source/src/protocols/motifs/IdentifyLigandMotifs.hh

```diff
@@ -114,7 +117,8 @@ private:
 		std::string & pdb_name,
 		protocols::motifs::MotifLibrary & motifs,
 		core::scoring::ScoreFunctionOP scorefxn,
-		utility::vector1< utility::vector1< core::Size > > & motif_indices_list
+		utility::vector1< utility::vector1< core::Size > > & motif_indices_list,
+		utility::vector1< core::Size > & prot_pos_that_made_motifs
```

Comment:
const correctness helps with documentation. If you're not going to change a value that's passed by reference, it's best to pass it by `const &` to annotate that fact. If you're consistent in doing so, it makes the fact that you _are_ modifying prot_pos_that_made_motifs stick out more.